.gitignore

```diff
@@ -3,4 +3,5 @@ composer.lock
 docs
 vendor
 coverage
-.env
\ No newline at end of file
+.env
+/.phpunit.result.cache
```

Comment:
This should go in a global gitignore. I'll remove this one.

Reply:
Should it? It's not local setup specific but project.

Reply:
Should it? It's not local setup specific but project.